test/index.js

```diff
@@ -371,20 +393,51 @@ it('uses specified route prefix for assets', function (done) {
         }
     });
 
-    server.register({ register: Tv, options: { port: 0 } }, { routes: { prefix: '/test' } }, function (err) {
+    server.register([Inert, Vision, { register: Tv, options: { port: 0 } }], { routes: { prefix: '/test' } }, function (err) {
 
         expect(err).to.not.exist();
+        server.start(function (err) {
 
-        server.inject('/test/debug/console', function (res) {
+            expect(err).to.not.exist();
+            server.inject('/test/debug/console', function (res) {
 
-            expect(res.statusCode).to.equal(200);
+                expect(res.statusCode).to.equal(200);
 
-            var cssPath = 'href="' + res.request.path + '/css/style.css';
-            var jsPath = 'src="' + res.request.path + '/js/main.js';
-            expect(res.result).to.contain(cssPath);
-            expect(res.result).to.contain(jsPath);
-            done();
+                var cssPath = 'href="' + res.request.path + '/css/style.css';
+                var jsPath = 'src="' + res.request.path + '/js/main.js';
+                expect(res.result).to.contain(cssPath);
+                expect(res.result).to.contain(jsPath);
+                done();
 
+            });
         });
     });
 });
+
+it('expects that Inert and Vision are already registered', function (done) {
+
+    var server = new Hapi.Server();
+    server.connection();
+
+    server.route({
+        method: 'GET',
+        path: '/',
+        handler: function (request, reply) {
+
+            return reply('1');
+        }
+    });
+
+    server.register({ register: Tv, options: { port: 0 } }, function (err) {
+
+        expect(err).to.not.exist();
+        try {
```

Comment:
I think you can do `expect(server.start(function {}).to.throw()`

https://github.com/hapijs/code#throwtype-message